### PR TITLE
Add support for allowing stalker to be configured

### PR DESCRIPTION
**Pull request by WorksButNotTested**
A little commit to allow access to the extended properties for fine-tuning the stalker engine when calling its constructor.

### CODE REVIEW

**Comment by meme** [comment]
Please rebase when possible so I can review. Thanks.

**Comment by WorksButNotTested** [comment]
This should be good to go now.

File: frida-gum-sys/build.rs [hunk 1]
```diff
@@ -158,6 +165,27 @@ fn main() {
...
+        #[cfg(not(feature = "auto-download"))]
+        let builder = if std::env::var("DOCS_RS").is_ok() {
+            builder.include("include")
+        } else {
+            &mut builder
+        };
+
+        builder
+            .file("stalker_params.c")
+            .opt_level(3)
```
Comment (meme) [inline frida-gum-sys/build.rs:185]: Question here: doesn't bindgen do this automatically when we are in release mode?

Comment (WorksButNotTested) [reply frida-gum-sys/build.rs:185]: You mean the `opt_level(3)`? I'm not sure tbh. I copied the template used by the other files above in the `build.rs`. I guess if one is changed, then they should all be? Perhaps extracting a method could reduce the code duplication too?

**Comment by meme** [comment]
Thanks!

**Comment by WorksButNotTested** [comment]
Fab. Thanks.

**Pull request closed without merging**